### PR TITLE
Changes for OS X (Big Sur, Monterey)

### PR DESCRIPTION
**Pull request by xenophon61**

### CODE REVIEW

File: bin/znapzendzetup [hunk 1]
```diff
@@ -482,7 +482,7 @@ sub main {
     }
     print STDERR <<NOTE;
 NOTE: if you have modified your configuration, send a HUP signal
-(pkill -HUP znapzend) to your znapzend daemon for it to notice the change.
+(sudo launchctl kill -HUP system/org.znapzend) to your znapzend daemon for it to notice the change.
```
Comment (oetiker) [inline bin/znapzendzetup:485]: This text might now confuse non mac users, as launchctl is MacOS specific

**Review by oetiker — CHANGES_REQUESTED** [review]

File: bin/znapzendzetup (hunk 1 above)
Comment (xenophon61) [reply bin/znapzendzetup:485]: You're right. I'm afraid I don't know how to insert a conditional statement, to check. Like I said, I'm not a programmer.
TIA
Xen

Comment (oetiker) [reply bin/znapzendzetup:485]: you could replace the code with:

```perl
if ($^O eq 'darwin') {
   print STDERR <<NOTE
... 
NOTE
} else {
   print STDERR <<NOTE
...
NOTE
}
```

Comment (oetiker) [inline bin/znapzendzetup:485]:
```suggestion
(pkill -HUP znapzend) to your znapzend daemon for it to notice the change.
```

File: init/org.znapzend.plist.in [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	<key>EnvironmentVariables</key>
 	<dict>
 		<key>PATH</key>
-		<string>/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+			<string>/usr/local/zfs/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
```
Comment (oetiker) [inline init/org.znapzend.plist.in:8]:
```suggestion
		<string>/usr/local/zfs/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
```

**Review by oetiker — APPROVED** [review]

**Comment by oetiker** [comment]
thank you!

**Comment by xenophon61** [comment]
No, thank YOU!
With greetings from Athens,
Xen

**Pull request merged**